R/produce_purdy_pictures.R

```diff
@@ -53,7 +81,7 @@ produce_purdy_pictures <- function(model_list){
       height = 9
     }
 
-    ggsave(filename, plot = fig, height = height, width = 18, units = "cm", dpi = 300)
+    ggsave(filename, plot = fig, height = height, width = 3.25 * length(unique(pdat$group)) + 5, units = "cm", dpi = 300)
 
   }
 
```

Comment:
Part of me wonders if the 'right' way to do this would be to return a vector of filenames, and set the targets as `type = 'file'. That is what the targets manual suggests for files produced as a side effect. But, a part of me also doesn't really care.

Reply:
Yeah I've done that before, but CBF at this point. If we change the filenames we should implement that.